### DISCUSSION TITLE
How to combine requests for data accessors that live on the same endpoint?

### DISCUSSION THREAD

**Discussion by jpreed00** [dpguthrie/yahooquery#293] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a6fe25a7-54e9-45d2-8f6f-72c6bc1ace48)

Those instructions are on the module doc page here: https://yahooquery.dpguthrie.com/guide/ticker/modules/

How does one configure the module to combine the requests?

**Comment by dpguthrie** [comment, marked as answer]
https://yahooquery.dpguthrie.com/guide/ticker/modules/#multiple-modules

**Comment by jpreed00** [comment]
Ahh, "get_modules", got it -- thanks!
